integration/simulation/network/dockerenclave.go

```diff
@@ -97,7 +97,9 @@ func (n *basicNetworkOfNodesWithDockerEnclave) TearDown() {
 	go func() {
 		for _, m := range n.ethNodes {
 			t := m
-			go t.Stop()
+			go func() {
+				_ = t.Stop()
```

Comment:
@otherview Same comment as above - maybe just suppress the linter here?

Reply:
Yep 👍